tsl/src/continuous_aggs/invalidation_plugin_cache.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+
+/*
+ * Cache for the hypertable invalidation log.
+ *
+ * This is used when processing the invalidations in the hypertable and
+ * consists of a set of invalidation entries similar to the hypertable
+ * invalidation log table. The main difference is that the the hypertable
+ * invalidation log table keeps a hypertable id, but the records returned here
+ * contains the relid. The reason for this is to avoid dependencies between
+ * dynamically loaded extensions (plugin and timescaledb extension).
```

Comment:
Again this connection to the hypertable invalidation log is confusing and also messy because of the mismatch in ID.

Also, IMO, this is a cache for the plugin, not the log (we want to get rid of the log, but this cache will probably remain).

```suggestion
 * Cache for the invalidations read by the plugin.
 *
 * The cache is used to accumulate invalidations for continuous aggregates based on 
 * changes to the source hypertable aggregated. When processing the WAL, 
 * multiple entries in the WAL might invalidate the same or overlapping ranges. The 
 * cache is used to merge those ranges into singular ranges until the cache is flushed
 * at the end of a transaction.
 *
 * The cache key is the source hypertable relid from which the 
 * invalidation was generated.
```